packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -1405,14 +1391,14 @@ class ThemeData with Diagnosticable {
   /// The color of [Material] when it is used as a [Card].
   final Color cardColor;
 
-  /// A set of twelve colors that can be used to configure the
-  /// color properties of most components.
+  /// A set of twelve colors that can be used to configure the color properties
```

Comment:
ColorScheme is 30 colors. It might make sense to macro the first line from https://api.flutter.dev/flutter/material/ColorScheme-class.html to avoid having this go out of sync again when additional color scheme colors are added in the future.

Reply:
OK, I'll update that.